andela/the_spinners-frontend
# #169817515 Add booking accommodation

#### What does this PR do?
Add booking accommodation
#### Description of Task to be completed?
- Add listing of accommodations
- Add displaying specific(selected) accommodation with all details
- Add listing rooms for specific accommodation
- Add comments section for specific accommodation
- Add tests for this feature
#### How should this be manually tested?
- Navigate to trip requests, click book accommodation on one of approved trip request
- You'll get a list of accommodation with selected one according to your destination
- Click book now on the particular room from the room list
- Fill the check-in and check-out and click book now
#### What are the relevant pivotal tracker stories?
#169817515
#### Screenshots 
<img width="1440" alt="Screen Shot 2020-03-12 at 23 01 49" src="https://user-images.githubusercontent.com/52411901/76566828-44d72f80-64b6-11ea-9adc-6c90b2168b0d.png">
